### DISCUSSION TITLE
Teslalogger on Raspberry pi zero 2w?

### DISCUSSION THREAD

**Discussion by Tobor7** [bassmaster187/TeslaLogger#846] [Q&A]
I wonder if anyone have try Teslalogger on a Raspberry pi zero 2w? The zero 2w is almost as performant as a Raspberry pi 3, but with less memory (512 MB for the zero VS 1 GB for the Raspberry pi 3). Can the smaller memory be a problem? Also, can Teslalogger have issue running over wifi instead of a wired lan?

**Comment by bassmaster187** [comment]
I try it with the old pi zero and it was very slow. 
During restore I made some tricks to save some memory, because even a Pi3 run into OutOfMemory Exception during restore process.

mybe it will work, but not pretty well.

![image](https://user-images.githubusercontent.com/6816385/181244272-7f64d320-46d4-463c-8a37-d6df5361eea2.png)
